.github/workflows/windows.yml

```diff
@@ -11,24 +11,33 @@ jobs:
   build:
     runs-on: "ubuntu-20.04"
     steps:
-      - uses: actions/setup-go@v1
-        with:
-          go-version: "1.18.3"
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
+
+      - name: Get GOVERSION content
+        id: goversion
+        run: echo ::set-output name=version::$(cat GOVERSION)
+
+      - uses: actions/setup-go@v1
+        with:
+          go-version: "${{ steps.goversion.outputs.version }}"
+
       - run: sudo apt install mingw-w64
       - run: |
           echo -n $PSIPHON_CONFIG_KEY > ./internal/engine/psiphon-config.key
           echo $PSIPHON_CONFIG_JSON_AGE_BASE64 | base64 -d > ./internal/engine/psiphon-config.json.age
         env:
           PSIPHON_CONFIG_KEY: ${{ secrets.PSIPHON_CONFIG_KEY }}
           PSIPHON_CONFIG_JSON_AGE_BASE64: ${{ secrets.PSIPHON_CONFIG_JSON_AGE_BASE64 }}
-      - run: ./mk MINGW_W64_VERSION="9.3-win32" ./CLI/ooniprobe-windows
+
+      - run: ./mk EXPECTED_MINGW_W64_VERSION="9.3-win32" ./CLI/ooniprobe-windows
+
```

Comment:
This is wrong

Reply:
This change (from an experimental branch) will be introduced by a subsequent PR.